web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -77,11 +77,14 @@ export function ResultList<T>(props: ResultListProps<T>) {
   }
 
   useEffect(() => {
+    // This needs to happen directly in `useEffect` because it gives us a guarantee
+    // that activeIndex matches the activeItemRef.
+    // `false` - bottom of the element will be aligned to the bottom of the visible area of the scrollable ancestor
+    activeItemRef.current?.scrollIntoView(false);
```

Comment:
I don't love it but I guess it works. I suppose a more explicit solution would be to render each element with its own ref, but I'm not even sure what would be a good way to go about it.

The biggest downside of this solution for me is that it hinges on this implicit assumption that `activeItemRef` (a value which does not trigger a rerender on change) gets updated whenever `activeItemIndex` changes.

Reply:
I was considering switching to an array, but then we would have to recreate it when the items list changes. This probably should be done in another `useEffect` which would add some complexity. I decided to stay with `activeItemRef` for simplicity reasons.